bio/varlociraptor/preprocess-variants/wrapper.py

```
@@ -0,0 +1,14 @@
+import subprocess as sp
+
+log = snakemake.log_fmt_shell(stdout=False, stderr=True)
```

Ruff

Undefined name `snakemake`

(F821)